### PR TITLE
feat: Course outline - Update section card

### PR DESCRIPTION
**Pull request by vladislavkeblysh**
- Updated section card and tests

[2u-165](https://youtrack.raccoongang.com/issue/2U-615)

### CODE REVIEW

File: src/course-outline/CourseOutline.jsx [hunk 1]
```diff
@@ -119,6 +119,8 @@ const CourseOutline = ({ courseId }) => {
                           section={section}
                           onOpenHighlightsModal={handleOpenHighlightsModal}
                           onOpenPublishModal={openPublishModal}
+                          // TODO add handler in Add new subsection feature
+                          onClickNewSubsection={null}
```
Comment (ruzniaievdm) [inline src/course-outline/CourseOutline.jsx:123]:
```suggestion
                          onClickNewSubsection={() => ({})}
```

**Review by ruzniaievdm — CHANGES_REQUESTED** [review]

File: src/course-outline/section-card/SectionCard.scss [hunk 2]
```diff
@@ -1,12 +1,12 @@
 .section-card {
   @include pgn-box-shadow(1, "centered");
 
-  padding: 1.5rem;
+  padding: 1rem 1.5rem 1.5rem;
```
Comment (PKulkoRaccoonGang) [inline src/course-outline/section-card/SectionCard.scss:4]:
```suggestion
  padding: $spacer 1.5rem 1.5rem;
```

**Review by ruzniaievdm — APPROVED** [review]

File: src/course-outline/section-card/SectionCard.scss [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
   cursor: move;
   margin-bottom: 1.5rem;
 
   .section-card__content {
-    margin-top: 1.75rem;
+    margin-top: 1rem;
```
Comment (PKulkoRaccoonGang) [inline src/course-outline/section-card/SectionCard.scss:9]:
```suggestion
    margin-top: $spacer;
```

File: src/course-outline/section-card/SectionCard.scss (hunk 2 above)
Comment (vladislavkeblysh) [reply src/course-outline/section-card/SectionCard.scss]: Done

File: src/course-outline/section-card/SectionCard.scss (hunk 3 above)
Comment (vladislavkeblysh) [reply src/course-outline/section-card/SectionCard.scss]: Done

**Review by PKulkoRaccoonGang — APPROVED** [review]

**Review by wowkalucky — APPROVED** [review]
OK.

**Pull request merged**